moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -785,17 +785,17 @@ bool ServoCalcs::enforcePositionLimits(sensor_msgs::msg::JointState& joint_state
 
   for (auto joint : joint_model_group_->getActiveJointModels())
   {
-    for (std::size_t c = 0; c < original_joint_state_.name.size(); ++c)
+    for (std::size_t c = 0; c < joint_state.name.size(); ++c)
     {
       // Use the most recent robot joint state
-      if (original_joint_state_.name[c] == joint->getName())
+      if (joint_state.name[c] == joint->getName())
```

Comment:
It makes sense to use Servo's internal join position here, rather than `original_joint_state_`. We do the same thing for velocity at L809  :+1: